### ISSUE TITLE
Discovery channels 

### ISSUE THREAD

**Issue by wallyps** [yt-dlp/yt-dlp#5574]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

US

### Provide a description that is worded well enough to be understood

Using 2022.11.17.40298

A week ago I started receiving messages that the software stating:

ERROR: [FoodNetwork] guys-grocery-games-food-network/fieri-thanksgiving-party: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/ytdl-patched/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

t-dlp --cookies-from-browser Chrome --hls-prefer-native --ap-mso Charter_Direct --ap-user user--ap-password password  --merge-output-format mkv --add-metadata --hls-prefer-native --write-sub --convert-subs srt -f "bestvideo[height<=720]+bestaudio/best[height<=720]"  -o test.mp4 "https://watch.foodnetwork.com/video/guys-grocery-games-food-network/fieri-thanksgiving-party"

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19043-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2020-11-04-git-cfdddec0c8-full_build-www.gyan.dev, ffprobe 2020-11-04-git-cfdddec0c8-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[Cookies] Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\wally\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\wally\AppData\Local\Google\Chrome\User Data\Local State"
[Cookies] Extracted 3501 cookies from chrome
[debug] cookie version breakdown: {'v10': 3501, 'other': 0, 'unencrypted': 0}
[debug] Proxy map: {}
[debug] Loaded 1724 extractors
[debug] Fetching release info: https://api.github.com/repos/ytdl-patched/yt-dlp/releases/latest
Latest version: 2022.11.17.40298, Current version: 2022.11.17.40298
yt-dlp is up to date (2022.11.17.40298)
[debug] [FoodNetwork] Extracting URL: https://watch.foodnetwork.com/video/guys-grocery-games-food-network/fieri-thanksgiving-party
[debug] Using fake IP 6.219.114.147 (US) as X-Forwarded-For
[FoodNetwork] guys-grocery-games-food-network/fieri-thanksgiving-party: Downloading JSON metadata
ERROR: [FoodNetwork] guys-grocery-games-food-network/fieri-thanksgiving-party: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/ytdl-patched/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 677, in extract
  File "yt_dlp\extractor\dplay.py", line 367, in _real_extract
  File "yt_dlp\extractor\dplay.py", line 78, in _get_disco_api_info
  File "yt_dlp\extractor\common.py", line 1037, in download_content
  File "yt_dlp\extractor\common.py", line 1001, in download_handle
  File "yt_dlp\extractor\common.py", line 871, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 828, in _request_webpage

  File "yt_dlp\extractor\common.py", line 810, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3705, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by Mr-Jake** [comment]
Unfortunately, it seems Discovery is changing formats. There is chance you may get m3u8 HLS, mpd DASH, or widevine protected video. There is no consistency and it seems to revert after some time. yt-dlp will only work with HLS and non-drm DASH.

**Comment by bashonly** [comment]
@Mr-Jake Maybe that's the case generally, but this particular issue doesn't have any indication of being DRM-related (yet).

@wallyps Do you have a Discovery+ account? What kind of cookies are you trying to pass?

None of the Discovery/dplay extractors currently have support for AdobePass (`--ap-mso`, etc). In the browser, most of the `watch.foodnetwork.com` content is provider-locked. Is this new? yt-dlp has only been using Discovery+ auth for the dplay extractors.

**Comment by pukkandan** [comment]
Why's the first line of log missing from OP?

**Comment by Mr-Jake** [comment]
I am experiencing this 403 error for the first time. I am properly passing the cookie. Videos I downloaded this morning are no longer downloading tonight. DiscoveryPlus works, but all Discovery GO networks return 403.

```
[debug] Command-line config: ['https://go.discovery.com/video/gold-rush-white-water-discovery/fist-full-of-nuggets', '--cookies', 'C:\\youtube-dl\\cookies.txt', '-F', '-v']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8 (No VT), error utf-8 (No VT), screen utf-8 (No VT)
[debug] yt-dlp version 2022.10.04 [4e0511f] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-7-6.1.7601-SP1
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.1.2-full_build-www.gyan.dev (setts), ffprobe 5.1.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24,mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1690 extractors
[debug] [GoDiscovery] Extracting URL: https://go.discovery.com/video/gold-rush-white-water-discovery/fist-full-of-nuggets
[debug] Using fake IP *removed* (US) as X-Forwarded-For
[GoDiscovery] gold-rush-white-water-discovery/fist-full-of-nuggets: Downloading JSON metadata
ERROR: [GoDiscovery] gold-rush-white-water-discovery/fist-full-of-nuggets: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 672, in extract
  File "yt_dlp\extractor\dplay.py", line 367, in _real_extract
  File "yt_dlp\extractor\dplay.py", line 78, in _get_disco_api_info
  File "yt_dlp\extractor\common.py", line 1032, in download_content
  File "yt_dlp\extractor\common.py", line 996, in download_handle
  File "yt_dlp\extractor\common.py", line 866, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 823, in _request_webpage

  File "yt_dlp\extractor\common.py", line 805, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3682, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default 
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by Mr-Jake** [comment]
Perhaps this will be helpful:

I generate my cookie file with the firefox extension cookies.txt.

In my browser, if I logout, log back in, and then generate a new cookie, I get the 403 error with yt-dlp.

However, I found an old cookie file I created about a month ago. I used that with yt-dlp and it worked. It downloaded as expected with no 403 error. Not sure how much longer it will work since the cookie will eventually expire. I will try to analyze the cookies later and see how they differ.
